src/main/java/com/parallax/server/blocklyprop/rest/RestProject.java

```diff
@@ -45,19 +63,36 @@
 @Group(name = "/project", title = "Project management")
 @HttpCode("500>Internal Server Error,200>Success Response")
 public class RestProject {
-    // Get a logger instance
+
+    /**
+     * Get a logger instance
+     */
     private static final Logger LOG = LoggerFactory.getLogger(RestProject.class);
 
-    // Connector to project services object
+
+    /**
+     * Connector to project services object
+     */
     private ProjectService projectService;
     
-    // Connector to project converter object
+
+    /**
+     * Connector to project converter object
+     */
     private ProjectConverter projectConverter;
 
 
+    /**
+     * Limit the number of records that can be returned in list functions
+     */
+    final int REQUEST_LIMIT = 100;
```

Comment:
Project listings are now limited to 100 records. This is hard coded only because it is not expected to change for the foreseeable future. The calling client is expected to iterate over the selection set.